src/Migration/RedisMigrationStorage.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using RapidCore.Migration;
+using StackExchange.Redis;
+
+namespace RapidCore.Redis.Migration
+{
+    public class RedisMigrationStorage : IMigrationStorage
+    {
+        private IDatabase GetDatabase(IMigrationContext context)
+        {
+            return context.Container.Resolve<IConnectionMultiplexer>().GetDatabase();
+        }
+
+        private string GetKey(string migrationName)
+        {
+            return $"migrations:{migrationName}";
```

Comment:
It would probably be wise to "namespace" the keys we store in redis - e.g `__rapidcore:migrations:{name}`